syntax = "proto3";

package camera;

message image{
    int32 width = 1;
    int32 height = 2;
    bytes imageData = 3;
}

message subscribeRequest{}

service imageStream{
    rpc getImage (subscribeRequest) returns (image){}
}

//protoc -I=(local do arquivo) --python_out=(locla de saida) (nome do arquivo .proto)
